packages/gui/src/electron/main.tsx

```diff
@@ -639,20 +652,29 @@ if (!handleSquirrelEvent()) {
         const filePath: string = path.join(thumbCacheFolder, request.url.replace(/^cached:\/\//, ''));
         callback({ path: filePath });
       });
-      mainWindow?.webContents.executeJavaScript('localStorage.getItem("cacheLimitSize");', true).then((stringValue) => {
-        try {
-          cacheLimitSize = stringValue ? JSON.parse(stringValue) : cacheLimitSize;
-        } catch (e) {
+      const prefs = readPrefs();
+      if(prefs['cacheLimitSize'] !== undefined){
+        try{
+          const prefs_cacheLimitSize = +prefs['cacheLimitSize'];
```

Comment:
Is the `+` a typo?

Reply:
This `+` is making sure that the value is to be a `number`.
```javascript
let str = "123";
+str === 123; // true
```